src/login/logind-session-device.c

```diff
@@ -104,6 +105,21 @@ static void sd_eviocrevoke(int fd) {
         }
 }
 
+static void sd_hidiocrevoke(int fd) {
+        static bool warned = false;
+
+        assert(fd >= 0);
+
+        if (!warned && ioctl(fd, HIDIOCREVOKE, NULL) < 0) {
+                if (errno == EINVAL) {
+                        log_warning_errno(errno, "Kernel does not support hidraw-revocation: %m");
+                        warned = true;
+                } else if (errno != ENODEV) {
+                        log_warning_errno(errno, "Failed to revoke hidraw device: %m");
```

Comment:
"Failed to revoke hidraw device, ignoring:"

Reply:
done, thanks